example-jobs/checkmarx-security-job-example.groovy

```diff
@@ -0,0 +1,18 @@
+import jenkins.automation.utils.CheckmarxSecurityJobBuilder
+
+def projectName ='foo'
+new CheckmarxSecurityJobBuilder(
+        name: "${projectName}-checkmarx",
+        description: "Sample checkmarx security job",
+        useOwnServerCredentials: false,
+        serverUrl: "http://awe-codescan-w-d01",
```

Comment:
Environmental variables sound like a good idea here, since all jobs use the same server Url and Group ID's